packages/internal/src/callback.js

```diff
@@ -73,6 +78,8 @@ harden(makeSyncFunctionCallback);
  * @returns {Callback<I>}
  */
 export const makeFunctionCallback = (target, ...bound) => {
+  isObject(target) ||
+    Fail`function callback target must be a function presence: ${target}`;
```

Comment:
If it's not done already, I'd prefer also allowing near functions (i.e. `typeof target === 'function'`), since `E` accepts them as a target.

Reply:
`isObject(target) === true ` for `typeof target === 'function'`

Reply:
`isObject` returns true for function input, and that case is covered by the first assertion in `test('isCallback', …)`.